client/src/hooks/useCreateProject.js

```diff
@@ -25,7 +25,58 @@ export const useCreateProject = () =>{
             setIsLoading(false);
             setError(json.error);
         }
+
+        if(response.ok) {
+            setIsLoading(false);
+            return json;
+        }
         
     }
-    return { createProject, isLoading, error }
+    const editProject = async (project, currentProjectId) => {
+        
+        const response = await fetch('api/homeowner/listings/'+currentProjectId, {
+            method: "PATCH",
+            headers: {
+                "Authorization": `Bearer ${user.token}`, 
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify(project)
+        });
+
+        const json = await response.json();
+
+        if(!response.ok){
+            setIsLoading(false);
+            setError(json.error);
+        }
+
+        if(response.ok) {
+            setIsLoading(false);
+            return json;
+        }
+    }
+
+    const deleteProject = async (currentProjectId) => {
+        
+        const response = await fetch('api/homeowner/listings/'+currentProjectId, {
```

Comment:
Please consider using a template literal instead of string concatenation: `/api/homeowner/listings/${currentProjectId}`